notes/adr/0041-select-star-dup-cols.adoc

```diff
@@ -0,0 +1,91 @@
+= 41. SQL SELECT * / Duplicate Columns
```

Comment:
pedantry: adoc idiom is [one sentence per line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) :slightly_smiling_face: